encoding.go

```diff
@@ -86,6 +88,8 @@ func ParseNode(serializedNode []byte, depth byte) (VerkleNode, error) {
 		return CreateInternalNode(serializedNode[internalBitlistOffset:internalCommitmentOffset], serializedNode[internalCommitmentOffset:], depth)
 	case eoAccountType:
 		return parseEoAccountNode(serializedNode, depth)
+	case singleSlotType:
```

Comment:
Add bullet to method's comment. (Be aware that base branch has a similar comment)